nano/nano_node/entry.cpp

```diff
@@ -55,13 +55,23 @@ class address_library_pair
 	bool operator< (const address_library_pair & other) const;
 	bool operator== (const address_library_pair & other) const;
 };
+
+constexpr std::size_t OPEN_FILE_DESCRIPTORS_LIMIT = 16384;
 }
 
 int main (int argc, char * const * argv)
 {
 	nano::set_umask (); // Make sure the process umask is set before any files are created
 	nano::logger::initialize (nano::log_config::cli_default ());
 
+	nano::set_file_descriptor_limit (OPEN_FILE_DESCRIPTORS_LIMIT);
+	auto const file_descriptor_limit = nano::get_file_descriptor_limit ();
+	nano::default_logger ().info (nano::log::type::daemon, "File descriptors limit: {}", file_descriptor_limit);
```

Comment:
This shouldn't be moved, this won't log anything.

Reply:
We could split setting the file descriptor limit with logging the warning about it being too low, though setting the file descriptor limit was moved here since it was only set when called with "--daemon" before and it needed to be set at least for "--migrate_database_lmdb_to_rocksdb" if not other commands when used with rocksdb when it has many files.

Reply:
Setting the limit here is not a problem, it's a good place for it. The problem is trying to log when logging is still in CLI mode. CLI commands usually have a very specific output format and to avoid polluting it with node details, only critical errors are logged. The warning probably needs to be logged via cerr, and the `"File descriptors limit: {}"...` line moved back into the daemon code, so it leaves a trace in log files if we ever need to debug it. 
 
```
nano::log_config nano::log_config::cli_default ()
{
	log_config config{};
	config.default_level = nano::log::level::critical;
	config.console.colors = false; // to avoid printing warning about cerr and colors
	config.console.to_cerr = true; // Use cerr to avoid interference with CLI output that goes to stdout
	config.file.enable = false;
	return config;
}
```

Reply:
I pushed a commit which should fix the logging. Also, as setting the limit was moved away from daemon class, targets other than cli wallet would no longer set it.